browser.go

```diff
@@ -214,11 +220,11 @@ func (browser *Browser) IsAndroid() bool {
 }
 
 func (browser *Browser) IsIPhone() bool {
-	return browser.Platform == "iOS" && browser.DeviceName == "iPhone"
+	return browser.Platform == "iOS" && browser.DeviceCodeName == "iPhone"
```

Comment:
Same as below, I think this should still be 
return browser.Platform == "iOS" && browser.DeviceName == "iPhone"

Reply:
This change is there because browser.DeviceName used to take value from DeviceCodeName line in the INI file (which I believe is wrong and misleading). For iPhone and iPad these lines seem to be the same, but I don't believe it's correct to expect that DeviceName should always match DeviceCodeName since they are technically different things.

Reply:
My take is that DeviceName is more semantically accurate here.  (I don't think "iPhone" is a code-name anymore :-). )  As you point out, in the current INI file the device-name and device-code-name area always the same for iPhone and iPad, so it doesn't make a difference as far as the code is concerned, it's just which makes more semantic sense.